### ISSUE TITLE
Homilies by John Chrysostom on John the Baptist

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2093]
I think we have too many records for homilies by John Chrysostom on John the Baptist:
LIT2085OntheL: This corresponds to EAE III, 289a, "On the Life and Decollation of John the Baptist"
LIT2125OntheG: This corresponds to EAE III, 289a, "On the Glory of John the Baptist"
LIT1623Homily: This refers originally to EAE III 294a, "On St John the Baptist" - I don't have access now to the reference given (ProvRec 29), but the manuscript given as witness, BLorient689, is also listed among the witnesses of LIT2085OntheL in the preceding article (it contains only one text related to John the Baptist).  This has been used in BM for BAVet32, f. 1-10, which is listed among the witnesses of LIT2125OntheG in the previous article.
I would suggest to delete LIT1623Homily (or to keep it, after disambiguation, as a general record for the potentially existing "third homily" according to EAE III, 289b or for unidentified homilies by John Chrysostom on John the Baptist - but since until now there was no need for it, maybe not identifying does not need to be encouraged) and update the manuscript description.
@DenisNosnitsin1970 @eu-genia

**Comment by eu-genia** [comment]
we need best to know exactly how many homilies on JB by JC exist in Ethiopic, if only two then OK to delete, if more then we should keep it, with a clear abstract. anyone we can ask?

**Comment by MarcinKrawczuk** [comment]
I'm terribly sorry to join the discussion without an invitation but perhaps we can use ProvRec (BTW I have it at home), pp. 59-85 as a type of clavis for the Ethiopian Chrysostomica, like e. g. Hammerschmidt's book for anaphoras?

In ProvRec there are two homilies on John the Baptist:
no. 17 - on decollation / Herodias, EMML 1763, f. 274-276 = **LIT2093OnHero**

no. 29 A - VEt 32, SPBK 61, EMML 6694 
no. 29 Aαβ - BL Orient 689, BnF d'Abb. 110, EMML 2514 and many others
(if my understanding is correct, these are two recensions, we use for both **LIT2085OntheL**)

Not in Proverbio we find:
Ethio-Spare MY-018, 44ra–60va = **LIT2125OntheG**

In conclusion, I would agree that LIT1623Homily could be deleted and some work is required on LIT2085OntheL.

**Comment by thea-m** [comment]
Thank you @MarcinKrawczuk ! And please do not apologise, but feel free to join any discussion - the only reason you were not mentioned was not to overwhelm you with emails, not to exclude your expertise.
That makes sense. I will wait until I get to the office to access the book and can work on the records then.
